Protocol.md

```diff
@@ -460,11 +461,11 @@ Frame Contents
                           Metadata & Request Data
 ```
 
-* __Frame Type__: (16) 0x06
-* __Flags__:
+* __Frame Type__: (6 bits = max value 63) 0x06
+* __Flags__: (10 bits)
     * (__M__)etadata: Metadata present
     * (__F__)ollows: More fragments follow this fragment.
-* __Initial Request N__: 32-bit signed integer representing the initial request N value for the stream. Only positive values are allowed.
+* __Initial Request N__: (32 bits = max value 2^31-1 = 2,147,483,647) Signed positive Integer representing the initial request N value for the stream. Value must be > 0.
```

Comment:
MUST is the standard for rules that are required. So yes we should be consistent for that.